textdb/textdb-dataflow/src/main/java/edu/uci/ics/textdb/dataflow/nlpextrator/NlpExtractor.java

```diff
@@ -136,7 +136,6 @@ public ITuple getNextTuple() throws Exception {
             }
             ITuple returnTuple = Utils.getSpanTuple(sourceTuple.getFields(),
                     spanList, returnSchema);
-            sourceTuple = sourceOperator.getNextTuple();
```

Comment:
@sam0227 : please confirm if this is indeed a bug in your old code.  We think so and will do the merge.  Still want to get your opinion.  Thank you.